tools/azure-sdk-tools/ci_tools/environment_exclusions.py

```diff
@@ -228,6 +228,7 @@
     "azure-eventhub-checkpointstoreblob",
     "azure-eventhub-checkpointstoreblob-aio",
     "azure-eventhub-checkpointstoretable",
+    "azure-identity",
```

Comment:
Something weird happened when merging with upstream here. Will double check that this is currently included on the list.

Reply:
Fixed